lib/siwapp_web/live/live_helpers.ex

```diff
@@ -12,4 +12,24 @@ defmodule SiwappWeb.LiveHelpers do
     modal_opts = [id: :modal, return_to: path, component: component, opts: opts]
     live_component(SiwappWeb.ModalComponent, modal_opts)
   end
+
+  def checked?(checked, id) do
+    Enum.member?(checked, id)
+  end
+
+  def status(paid, due_date) do
+    cond do
+      paid -> :paid
+      pending?(due_date) -> :pending
+      true -> :past_due
+    end
+  end
+
+  def invisible?(checked) do
```

Comment:
instead of writing a fn which does just 1 thing, use the fn below and use it directly in the template